src/plugins/vis_type_timeseries/public/application/components/aggs/filter_ratio.js

```diff
@@ -101,7 +101,14 @@ export const FilterRatioAgg = (props) => {
               />
             }
           >
-            <EuiFieldText onChange={handleTextChange('numerator')} value={model.numerator} />
+            <QueryBarWrapper
+              query={{
+                language: model.numerator.language || getDefaultQueryLanguage(),
```

Comment:
nit: I don't see a case when when **model.numerator.language** will be undefined. If I understand properly after running your migration script all **saved objects** should contain language and query. For new visualizations you added logic in **defaults** constant.
But this comment not so critical for me.